contentgrid-spring-boot-autoconfigure/src/test/java/com/contentgrid/spring/boot/autoconfigure/data/audit/JpaAuditingAutoConfigurationTest.java

```diff
@@ -14,35 +15,48 @@
 import org.springframework.boot.test.context.FilteredClassLoader;
 import org.springframework.boot.test.context.runner.WebApplicationContextRunner;
 import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+import org.springframework.security.oauth2.jwt.JwtClaimAccessor;
 
 class JpaAuditingAutoConfigurationTest {
 
     private final WebApplicationContextRunner contextRunner = new WebApplicationContextRunner()
             .withConfiguration(AutoConfigurations.of(DataSourceAutoConfiguration.class,
                     TransactionAutoConfiguration.class, HibernateJpaAutoConfiguration.class,
-                    JpaAuditingAutoConfiguration.class
+                    JpaAuditingAutoConfiguration.class, JwtJpaAuditingAutoConfiguration.class
             ));
 
     @Test
     void checkDefaults() {
         contextRunner.withInitializer(ConditionEvaluationReportLoggingListener.forLogLevel(LogLevel.INFO))
                 .run(context -> {
                     assertThat(context).hasNotFailed();
-                    assertThat(context).hasSingleBean(JpaAuditingConfiguration.class);
                     assertThat(context).hasSingleBean(JwtAuditorAware.class);
+                    assertThat(context).doesNotHaveBean(DefaultAuditorAware.class);
                     assertThat(context).hasSingleBean(AuditingEntityListener.class);
                 });
     }
 
     @Test
-    void checkWithoutDependency() {
+    void checkWithoutUserMetadata() {
         contextRunner.withInitializer(ConditionEvaluationReportLoggingListener.forLogLevel(LogLevel.INFO))
-                .withClassLoader(new FilteredClassLoader(JpaAuditingConfiguration.class))
+                .withClassLoader(new FilteredClassLoader(UserMetadata.class))
                 .run(context -> {
                     assertThat(context).hasNotFailed();
-                    assertThat(context).doesNotHaveBean(JpaAuditingConfiguration.class);
                     assertThat(context).doesNotHaveBean(JwtAuditorAware.class);
-                    assertThat(context).doesNotHaveBean(AuditingEntityListener.class);
+                    assertThat(context).doesNotHaveBean(DefaultAuditorAware.class);
+                    assertThat(context).hasSingleBean(AuditingEntityListener.class);
+                });
+    }
+
+    @Test
+    void checkWithoutJwtClaimAccessor() {
+        contextRunner.withInitializer(ConditionEvaluationReportLoggingListener.forLogLevel(LogLevel.INFO))
```

Comment:
if you are using the `ConditionEvaluationReportLoggingListener` everywhere anyway, you should probably assign it on the shared `contextRunner` instance ?